Task_2/mlcubes/model/README.md

```diff
@@ -0,0 +1,209 @@
+```
+
+## How to modify this project
+
+You can change each file described above in order to add your own implementation. In case you need more information on the internals of MLCube, check out the official [git repository](https://github.com/mlcommons/mlcube) or [documentation](https://mlcommons.github.io/mlcube/).
+
+<details><summary><b>Requirements file </b></summary>
+<p>
+
+In this file (`requirements.txt`) you can add all the python dependencies needed for running your implementation. These dependencies will be installed during the creation of the docker image, which happens automatically when you run the ```mlcube run ...``` command.
+</p>
+</details>
+
+<details><summary><b>Dockerfile </b></summary>
+<p>
+
+This file can be adapted to add your own docker labels, install some OS dependencies or to change the base docker image. Note however that we *strongly recommend* to use one of our proposed base images (`nvcr.io/nvidia/pytorch:20.08-py3` or tensorflow equivalent), to make sure your application can be executed in the federated evaluation. Note that the [pytorch (or tensorflow) version](https://docs.nvidia.com/deeplearning/frameworks/support-matrix/index.html) inside this container is 1.7.0 (or 2.2.0), so for inference you may not be able to use features introduced in later versions, unfortunately.
+
+</p>
+</details>
+
+<details><summary><b>MLCube yaml file </b></summary>
+<p>
+
+`mlcube.yaml` contains instructions about the docker image and platform that will be used, information about the project (name, description, authors), and also the tasks defined for the project. **Note** that this file is not submitted and changes will hence not have any effect in the official evaluation. We will use the provided template with the name of your docker image instead.
+
+In the existing implementation you will find the `infer` task, which will be executed in the federated evaluation. It takes the following parameters:
+
+- Input parameters:
+  - data_path: folder path containing input data
+  - checkpoint_path: folder path containing model checkpoints
+  - parameters_file: Extra parameters
+- Output parameters:
+  - output_folder: folder path where output data will be stored
+
+This task loads the input data, processes it and then saves the output result in the output_folder. It also prints some information from the extra parameters.
+
+</p>
+</details>
+
+<details><summary><b>MLCube python file </b></summary>
+<p>
+
+The `mlcube.py` file is the handler file and entrypoint described in the dockerfile. Here you can find all the logic related to how to process each MLCube task. For most challenge participants, the provided template should be usable without modifications.
+If you want to add a new task first you must define it inside the `mlcube.yaml` file with its input and output parameters and then you need to add the logic to handle this new task inside the `mlcube.py` file.
```

Comment:
This is correct, but is it really necessary for the competitors? Since the mlcube.yaml file won't be submitted, we shouldn't encourage people adding tasks that can't be passed through

Reply:
Thinking through this, I'm guessing this was added so competitors add a training task. Is that correct? If so then this is fine, although it would be good to provide that use-case example, and to reiterate that those changes won't be accessible after submission

Reply:
Good points, I clarified that usually participants should not make new tasks.